src/main/webapp/showMessage.jsp

```diff
@@ -19,7 +19,7 @@
                     }
                 }
             }
-            var nextUrl = "<%=Const.ActionURIs.INSTRUCTOR_EVALS_PAGE%>";
+            var nextUrl = "<%=Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE%>";
             if(user!==null){
```

Comment:
While I have modified this file and it gets tested in AllAccessControlUiTest, it does not seem to be used anywhere. Will make a new issue for this